### PR TITLE
fixed tooltip float number

### PR DESCRIPTION
**Pull request by viniciusdacal**
@caruccio

### CODE REVIEW

**Comment by viniciusdacal** [comment]
resolves #4

File: src/components/DetailedChart/DetailedChart.js [hunk 1]
```diff
@@ -70,7 +69,7 @@ const DetailedChart = ({
               dot={false}
               activeDot={{r: 8}}
               unit={unit}
-              formatter={(average) => parseInt(average)}
+              formatter={(average) => parseFloat(average).toFixed(1)}
```
Comment (caruccio) [inline src/components/DetailedChart/DetailedChart.js]: Can we have this as a prop to DetailedChart?

Comment (viniciusdacal) [reply src/components/DetailedChart/DetailedChart.js]: indeed, I created a props called `averageFormatter` to support this :)

**Comment by caruccio** [comment]
LGTM

**Pull request merged**